packages/openssh/build.sh

```diff
@@ -2,12 +2,18 @@ TERMUX_PKG_HOMEPAGE=https://www.openssh.com/
 TERMUX_PKG_DESCRIPTION="Secure shell for logging into a remote machine"
 TERMUX_PKG_LICENSE="BSD"
 TERMUX_PKG_MAINTAINER="Joshua Kahn @TomJo2000"
-TERMUX_PKG_VERSION="9.9p2"
+TERMUX_PKG_VERSION="10.0p2"
 TERMUX_PKG_SRCURL=https://github.com/openssh/openssh-portable/archive/refs/tags/V_$(sed 's/\./_/g; s/p/_P/g' <<< $TERMUX_PKG_VERSION).tar.gz
-TERMUX_PKG_SHA256=082dffcf651b9db762ddbe56ca25cc75a0355a7bea41960b47f3c139974c5e3e
+TERMUX_PKG_SHA256=a25b32645dc6b474064b9deb07afc9d8e37b127d026a1170b54feb929145140c
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_DEPENDS="krb5, ldns, libandroid-support, libedit, openssh-sftp-server, openssl, termux-auth, zlib"
+TERMUX_PKG_RECOMMENDS="termux-services"
 TERMUX_PKG_CONFLICTS="dropbear"
+
```

Comment:
nitpick: extra newline

Reply:
I think that was intentional because of the comment, but I moved that around afterwards so I may have added an extra one.